docs/sources/image.md

```diff
@@ -0,0 +1,97 @@
+# Image source
+
+## Summary
+
+The image source provides the contents in a container image as the source of the bundle.  The `source.type` for the image source is `image`.
+When creating a image source, a reference to an URL of the image is specified. It is expected that a proper format of bundle contents is present
+in the image.
```

Comment:
Are these docs meant to be descriptions of our implementations of the various source types, or are they meant to be specs for _how_ source implementations must work? It's probably important to distinguish in the docs so that readers can understand.

Generally, I'd expect most provisioners to vendor and use the source implementations from this repo, but technically a provisioner has some freedom in its implementation of the source API (barring any conformance tests we might come up with).

Reply:
I see.  That's a good point.  I was focusing on the former.  Do we create some docs for provisioner providers?